lib/setuid.js

```diff
@@ -0,0 +1,15 @@
+var Config = require("./config");
+
+if (Config.get("setuid.enabled")) {
+    setTimeout(function() {
```

Comment:
How did you determine the timeout?  My concern would be that if the port bind happened to take uncharacteristically long, it could be a tricky nondeterministic issue to debug.

The ideal solution would be to add the callback to `http.createServer` etc. so that the setuid happens after the port finishes binding.  I'm not sure if there's a clean way to do this for multiple servers, besides keeping track of a counter variable or something.